configuration.rst

```diff
@@ -30,28 +30,29 @@ The user specified in the db-uri is also known as the authenticator role. For mo
 
 Here is the full list of configuration parameters.
 
-====================  ======  =========  ========
-Name                  Type    Default    Required
-====================  ======  =========  ========
-db-uri                String             Y
-db-schema             String             Y
-db-anon-role          String             Y
-db-pool               Int     10
-db-pool-timeout       Int     10
-db-extra-search-path  String  public
-server-host           String  !4
-server-port           Int     3000
-server-unix-socket    String
-server-proxy-uri      String
-jwt-secret            String
-jwt-aud               String
-secret-is-base64      Bool    False
-max-rows              Int     ∞
-pre-request           String
-app.settings.*        String
-role-claim-key        String  .role
-raw-media-types       String
-====================  ======  =========  ========
+====================    ======  =========  ========
+Name                    Type    Default    Required
+====================    ======  =========  ========
+db-uri                  String             Y
+db-schema               String             Y
+db-anon-role            String             Y
+db-pool                 Int     10
+db-pool-timeout         Int     10
+db-extra-search-path    String  public
+server-host             String  !4
+server-port             Int     3000
+server-unix-socket      String
+server-unix-socket-mode String  755        
+server-proxy-uri        String
+jwt-secret              String
+jwt-aud                 String
+secret-is-base64        Bool    False
+max-rows                Int     ∞
+pre-request             String
+app.settings.*          String
+role-claim-key          String  .role
+raw-media-types         String
+====================    ======  =========  ========
```

Comment:
Somehow the table is not rendering for me locally. Try formatting it like:
```suggestion
=======================  ======  =========  ========
Name                     Type    Default    Required
=======================  ======  =========  ========
db-uri                   String             Y
db-schema                String             Y
db-anon-role             String             Y
db-pool                  Int     10
db-pool-timeout          Int     10
db-extra-search-path     String  public
server-host              String  !4
server-port              Int     3000
server-unix-socket       String
server-unix-socket-mode  String  755
server-proxy-uri         String
jwt-secret               String
jwt-aud                  String
secret-is-base64         Bool    False
max-rows                 Int     ∞
pre-request              String
app.settings.*           String
role-claim-key           String  .role
raw-media-types          String
=======================  ======  =========  ========
```

Reply:
I totally forgot that. I hope now it works.

Reply:
Looking good now.